src/utilities/json.ts

```diff
@@ -21,5 +21,11 @@ export function parseUnknownJson(value: unknown): unknown {
 export function stringifyUnknownJson(value: unknown): string | null | undefined {
   const parsed = parseUnknownJson(value)
 
+  return stringify(parsed)
+}
+
+export function formatUnknownJson(value: unknown): string {
```

Comment:
These two methods are the same except the return type. Seems like a red flag? They cannot both be correct right?

Reply:
oof, one day i'll explain my thought process here but it's embarrassing; removed in favor of the existing one